geode-core/src/main/java/org/apache/geode/management/internal/configuration/mutators/RegionConfigManager.java

```diff
@@ -44,7 +47,11 @@ public void delete(BasicRegionConfig config, CacheConfig existing) {
   }
 
   @Override
-  public List<BasicRegionConfig> list(BasicRegionConfig config, CacheConfig existing) {
-    throw new NotImplementedException("Not implemented yet");
+  public List<RegionConfig> list(BasicRegionConfig filter, CacheConfig existing) {
```

Comment:
looks to me like this takes a BasicRegionConfig, not a CacheElement as you mentioned in previous comment on this PR, am I missing something?